src/frontend/src/flows/addDevice/welcomeView/registerTentativeDevice.ts

```diff
@@ -25,7 +25,7 @@ import { authenticatorAttachmentToKeyType } from "../../../utils/authenticatorAt
 export const registerTentativeDevice = async (
   userNumber: bigint,
   connection: Connection
-): Promise<void> => {
+): Promise<"ok"> => {
```

Comment:
not sure, it isn't `"ok" | never`?

Reply:
I think `"ok" | never` is pretty much the same as `"ok"` right?

Reply:
guess so, just makes it explicit for the reader as well I guess.

Reply:
I think it would make it slightly less explicit actually; `never` suggests the function never returns (i.e. there's an infinite loop or something). At least that's how I interpret it, but maybe that's just the Haskell way of looking at it 😅